docs/settings.md

```diff
@@ -125,7 +125,8 @@ Cookie: sessionid=xxxx
       "locale":"fr",
       "auto_update": true,
       "email": "alice@example.com",
-      "public_name":"Alice Martin"
+      "public_name":"Alice Martin",
+      "auth_mode": "basic"
```

Comment:
Can you also update the auth documentation? https://cozy.github.io/cozy-stack/auth.html#faq and https://cozy.github.io/cozy-stack/auth.html#get-authlogin

Reply:
done https://github.com/cozy/cozy-stack/pull/1024/commits/7e1d6d4461789c6799cd4bf379fd93c96d8c0095